jenkins-scripts/dsl/brew_release.dsl

```diff
@@ -111,40 +111,37 @@ release_job.with
        pattern("${PR_URL_export_file_name}")
        allowEmpty()
      }
-
-     downstreamParameterized
-     {
-        trigger(bottle_builder_job_name)
-        {
-          condition('SUCCESS')
-          parameters {
-            currentBuild()
-            propertiesFile("${PR_URL_export_file_name}")
-          }
-        }
-     }
    }
 }
 
 // -------------------------------------------------------------------
 // 2. BREW bottle creation MATRIX job from pullrequest
 def bottle_job_builder = matrixJob(bottle_builder_job_name)
-OSRFOsXBase.create(bottle_job_builder)
+// set enable_github_pr_integration flag to false so we can customize trigger behavior
+OSRFBrewCompilationAnyGitHub.create(bottle_job_builder,
+                                    "osrf/homebrew-simulation",
+                                    false, [], false)
```

Comment:
It might sound a bit picky, but I tried to use constant to get some meaning to function arguments so they are easy to read and maintain. Something like https://github.com/ignition-tooling/release-tools/blob/master/jenkins-scripts/dsl/gazebo.dsl#L26-L27

Reply:
excellent approached, adopted in 0baa62b174851feb35725bf2376c55f754ef1ca2